apis/locale/locales/en-US.json

```diff
@@ -23,6 +23,10 @@
     "value": "Search in listbox",
     "comment": "Action text to search in listbox"
   },
+  "Menu.More": {
+    "value": "More",
+    "comment": "$"
```

Comment:
Suggested comment: `Menu option to display list of additional actions to choose from (ffd 20200520)`

Going forward I checked with the tech writer team and my understanding is that Dan Chabot (tech writer) will be available to review strings. I'll check directly with him and ask him to join this org.

Reply:
Maybe we should add that to the documentation https://github.com/qlik-oss/nebula.js/blob/master/apis/locale/locales/README.md

Reply:
Yes, I plan to update the readme next week, to capture everything ([RDL-3536](https://jira.qlikdev.com/browse/RDL-3536))